src/hermes/commands/workflow.py

```diff
@@ -207,27 +221,28 @@ def deposit(click_ctx: click.Context, initial, auth_token, file):
             _log.error(
                 f"Plugin name {plugin_name!r} is not unique within group {plugin_group!r}"
             )
-            click_ctx.exit(1)
     except ValueError:  # not enough values to unpack
         _log.error(f"Plugin name {plugin_name!r} was not found in group {plugin_group!r}")
-        click_ctx.exit(1)
+        sys.exit(1)
 
     # TODO: Could this raise an exception?
     deposit_plugin_class: BaseDepositPlugin = ep.load()
     deposit_plugin = deposit_plugin_class(click_ctx, ctx)
 
-    try:
-        deposit_plugin()
-    except (RuntimeError, MisconfigurationError) as e:
-        _log.error(f"Error in {plugin_group!r} plugin {plugin_name!r}: {e}")
-        click_ctx.exit(1)
+    for entry_point in loaded_entry_points:
+        try:
+            deposit_plugin(path, config_path, initial, auth_token, files, ctx)
+        except (RuntimeError, MisconfigurationError) as e:
+            _log.error(f"Error in {group!r} entry point {name!r}: {e}")
+            sys.exit(1)
```

Comment:
There's something wrong here. `loaded_entry_points` does not exist and neither do `group` and `name`.